packages/react-aria-components/docs/Disclosure.mdx

```diff
@@ -103,6 +106,108 @@ import {UNSTABLE_Disclosure as Disclosure, Button, UNSTABLE_DisclosurePanel as D
 
 </details>
 
+
+## Features
+
+Custom styled disclosures can be difficult to build in an accessible way with the correct aria pattern. When implementing, you must manually ensure that the button and panel are semantically connected via ids for accessibility. `Disclosure` helps with this and other accessibility features while allowing for custom styling.
+
+* **Flexible** - Structured such that it can be used standalone or combined with other disclosures to form a `DisclosureGroup`
+* **Accessible** - When focused, a disclosure's visibility can be toggled with either the <Keyboard>Enter</Keyboard> or <Keyboard>Space</Keyboard> key, and the appropriate aria attributes are automatically applied.
+* **Styleable** - Keyboard focus, disabled and expanded states are provided for easy styling. These states only apply when interacting with an appropriate input device, unlike CSS pseudo classes.
+
+## Anatomy
+
+<Anatomy />
```

Comment:
Missing the inline styles that other anatomy diagrams have (styles like `width:calc(100% - 64px)`)